ruby/README.md

```diff
@@ -645,15 +645,18 @@ result = hash.map { |k, v| v + 1 }
 result = hash.map { |_, v| v + 1 }
 ```
 
-### Do not use a hash literal to destructure keyword args.
+### Do not rely on a hash being destructured into keyword args.
```

Comment:
Let's mention Ruby 3:
```
### Do not rely on a hash being implicitly destructured into keyword args, because Ruby 3 will remove this implicit behavior. Instead, use `**`.